content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -112,11 +112,19 @@ that updates those files from a remote source, as in the following diagram:
 
 {{< figure src="/images/docs/pod.svg" alt="Pod creation diagram" class="diagram-medium" >}}
 
-Some Pods have {{< glossary_tooltip text="init containers" term_id="init-container" >}} as well as {{< glossary_tooltip text="app containers" term_id="app-container" >}}. Init containers run and complete before the app containers are started.
+Some Pods have {{< glossary_tooltip text="init containers" term_id="init-container" >}} as well as {{< glossary_tooltip text="app containers" term_id="app-container" >}}. By default, init containers run and complete before the app containers are started.
 
 Pods natively provide two kinds of shared resources for their constituent containers:
 [networking](#pod-networking) and [storage](#pod-storage).
 
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
```

Comment:
Maybe place this above the previous paragraph so its right after the discussion of regular init containers?

Reply:
This shortcode appears in the middle of a section, so it is gonna be confusing to readers. I'd suggest we drop this shortcode and clearly state that the SidecarContainers feature gate is in Alpha since version 1.28.
Later on, when the feature graduates, we can easily grep the feature name and adapt the text.